libs/agno/agno/run/messages.py

```diff
@@ -20,6 +21,12 @@ class RunMessages:
     user_message: Optional[Message] = None
     extra_messages: Optional[List[Message]] = None
```

Comment:
We should rename this class to LLMInput after this

Reply:
Will do!